paasta_tools/chronos_tools.py

```diff
@@ -720,3 +720,21 @@ def find_matching_parent_job(job_name):
         return sorted_jobs[0]['name']
     else:
         return None
+
+
+def compose_check_name_for_job(service, instance):
+    """Compose a sensu check name for a given job"""
+    return 'check-chronos-jobs.%s%s%s' % (service, INTERNAL_SPACER, instance)
```

Comment:
ah, this isn't the correct place for this.

Here's what I would do:

in ``chronos_tools.py`` and ``marathon_tools.py``:
```python
def compose_check_name_for_service_instance(check_name, service, instance):
      return '%s.%s%s%s' % (check_name, service, INTERNAL_SPACER, instance)
```

in ``check_chronos_jobs.py``

```python
def send_event(service, instance):
     """check_chronos_jobs specific wrapper for monitoring_tools.send_event_to_sensu"""
     check_name = chronos_tools.compose_check_name_for_job('check_chronos_jobs', service, instance)
     return monitoring_tools.send_event(check_name, service, instance....)
```

You can then follow this pattern of copying the ``check_chronos_tools.send_event`` function in other scripts (like in #198)